pywebpush/__init__.py

```diff
@@ -152,31 +152,33 @@ def encode(self, data, content_encoding="aesgcm"):
             raise WebPushException("Invalid content encoding specified. "
                                    "Select from " +
                                    json.dumps(self.valid_encodings))
-        if (content_encoding == "aesgcm"):
-            salt = os.urandom(16)
         # The server key is an ephemeral ECDH key used only for this
         # transaction
         server_key = ec.generate_private_key(ec.SECP256R1, default_backend())
-        crypto_key = base64.urlsafe_b64encode(
-            server_key.public_key().public_numbers().encode_point()
-        ).strip(b'=')
+        crypto_key = server_key.public_key().public_numbers().encode_point()
+        if (content_encoding == "aesgcm"):
+            salt = os.urandom(16)
+            keyid = "string"
```

Comment:
Ah, it appears that bug may have been fixed. There was a period where a keyid needed to be specified. It was a nonce, so any value could be used.